packages/nextjs-template/cypress/integration/home.ts

```diff
@@ -0,0 +1,5 @@
+describe("Index", () => {
+  it("successfully loads", () => {
+    cy.visit("/");
```

Comment:
i forget - does `cy.visit` crash if it can't load the page?

Reply:
It will throw a `CypressError` 

```cy.visit() failed trying to load:

http://localhost:3000/other

The response we received from your web server was:

  > 404: Not Found

This was considered a failure because the status code was not 2xx.

This http request was redirected '1' time to:

  - 308: http://localhost:3000/other/

If you do not want status codes to cause failures pass the option: failOnStatusCode: false```